src/modules/minio/minio.service.ts

```diff
@@ -120,4 +120,7 @@ export class MinioService {
       throw new InternalServerErrorException('Objects removal failed');
     }
   }
+  async generateFileUrl(fileName: string, bucketName = BucketName.Images) {
+    return await this.minioClient.presignedGetObject(bucketName, fileName);
```

Comment:
you should not use `presignedGetObject` to fetch images, Akilbek’s approach was correct. otherwise the urls with expire in 7 days. the reason urls didn’t work was because the link was https based instead of http